src/functions/frontend/rctToLft.m

```diff
@@ -1,4 +1,4 @@
-function lft_out = rctToLft(rct_obj) 
+function lft_out = rctToLft(rct_obj, varargin) 
```

Comment:
Also suggest changing `varargin` to a more meaningful name, especially since you're not expecting more than one variable here (correct?).

Reply:
Can do - was under the impression we had to use varargin for optional variables.

Reply:
No, in MATLAB, functions can be defined with variables that are not included in a function call.